templates/apply-damage/apply-damage-dialog.html

```diff
@@ -171,11 +171,28 @@ <h4>{{localize "GURPS.addTacticalRules"}}</h4>
             <div><label for='tactical-blunttrauma'>{{localize "GURPS.bluntTrauma"}}</label></div>
             <div class='pdflink'>B379</div>
 
-            <div><input id='tactical-armordivisor' type='checkbox' value='armordivisor' {{#if
-                CALC.useArmorDivisor}}checked{{/if}}></div>
-            <div><label for='tactical-armordivisor'>{{localize "GURPS.addArmorDivisor"}}</label></div>
-            <div class='pdflink'>B378</div>
+          </div>
+
+          <div class='shaded collapsible-wrapper options-drawer'>
+            <input id='tactical-armordivisor' class='toggle' type='checkbox' value='armordivisor' {{#if
+                   CALC.useArmorDivisor}}checked{{/if}}>
+            <label for='tactical-armordivisor' class='label-toggle'>{{localize "GURPS.addArmorDivisor"}}: <span
+                    class='pdflink'>B378</span></label>
+            <div class='grid grid-5col continuation collapsible-content'>
+              <label><input type='radio' name='tactical-armordivisor' value='2' {{#if (eq CALC.armorDivisor 2)}}
+                            checked{{/if}}>&nbsp;2</label>
+              <label><input type='radio' name='tactical-armordivisor' value='3' {{#if (eq CALC.armorDivisor 3)}}
+                            checked{{/if}}>&nbsp;3</label>
+              <label><input type='radio' name='tactical-armordivisor' value='5' {{#if (eq CALC.armorDivisor 5)}}
+                            checked{{/if}}>&nbsp;5</label>
+              <label><input type='radio' name='tactical-armordivisor' value='100' {{#if (eq CALC.armorDivisor 100)}}
```

Comment:
Shouldn't this value be 10? (from B102).
```suggestion
              <label><input type='radio' name='tactical-armordivisor' value='10' {{#if (eq CALC.armorDivisor 10)}}
```